Ruby-Project/student_class.rb

```diff
@@ -0,0 +1,11 @@
+require './person_class'
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] As mentioned in the previous review, here is what we are expected to do as seen below
```
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)

```